eth/eventhandler/handlers.go

```diff
@@ -567,7 +567,7 @@ func (eh *EventHandler) processClusterEvent(
 		}
 	}
 
-	return toUpdate, updatedPubKeys, nil
+	return operatorShares, operatorValidatorPubKeys, nil
```

Comment:
From the looks of it, `processClusterEvent` func does 2 things:
- updates shares
- returns a list of operator (own) shares & pub-keys

can we split this `func` in 2 then to make it clear these things aren't related ?

Reply:
I guess this applies to all `EventHandler` functions